Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -558,30 +559,69 @@ class IntegrationLogger(object):
 
     def __init__(self, ):
         self.messages = []  # type: list
-
-    def __call__(self, message):
+        self.write_buf = []  # type: list
+        self.replace_strs = []  # type: list
+        if (not os.getenv('COMMON_SERVER_NO_AUTO_REPLACE_STRS') and hasattr(demisto, 'getParam')
```

Comment:
need to add a comment here about `COMMON_SERVER_NO_AUTO_REPLACE_STRS`

Reply:
added a comment